ironfish/src/fileStores/config.ts

```diff
@@ -374,6 +388,8 @@ export class Config extends KeyStore<ConfigOptions> {
       networkId: DEFAULT_NETWORK_ID,
       customNetwork: '',
       maxSyncedAgeBlocks: 60,
+      memPoolMaxSizeBytes: 60000000,
+      memPoolRecentlyEvictedCacheSize: 60000,
```

Comment:
as mempool size is determined in bytes, should the cache be as well? then can add a downstream `sizeInTxns` that is determined from the size in bytes

Reply:
Yea could be a good idea. I don't wanna do it right now though since it complicates things and don't anticipate people really changing this config setting